.travis.yml

```diff
@@ -86,5 +83,10 @@ jobs:
     - env: PIP=master
     - python: 3.8-dev
 
+after_success:
+  - travis_retry pip install codecov coveralls
+  - codecov -n "py${TRAVIS_PYTHON_VERSION}-pip${PIP}-${TRAVIS_OS_NAME}"
```

Comment:
Could you fix it the same way in the `.appveyour.yml`?

Reply:
Done.  Sorry for missing it in the first place..